repo/packages/M/marathon/2/config.json

```diff
@@ -0,0 +1,391 @@
+          "type" : "integer"
+        },
+        "http-port" : {
+          "description" : "The port to listen on for HTTP requests",
+          "type" : "integer"
+        },
+        "http-realm" : {
+          "description" : "The security realm (aka 'area') associated with the credentials",
+          "type" : "string",
+          "default" : "Mesosphere"
+        },
+        "https-address" : {
+          "description" : "The address to listen on for HTTPS requests.",
+          "type" : "string"
+        },
+        "https-port" : {
+          "description" : "The port to listen on for HTTPS requests",
+          "type" : "integer"
+        },
+        "launch-token-refresh-interval" : {
+          "description" : "The interval (ms) in which to refresh the launch tokens to --launch_token_count",
+          "type" : "integer",
+          "default" : 30000
+        },
+        "launch-tokens" : {
+          "description" : "Launch tokens per interval",
+          "type" : "integer",
+          "default" : 100
+        },
+        "leader-proxy-connection-timeout" : {
+          "description" : "Maximum time, in milliseconds, to wait for connecting to the current Marathon leader from another Marathon instance.",
+          "type" : "integer",
+          "default" : 5000
+        },
+        "leader-proxy-read-timeout" : {
+          "description" : "Maximum time, in milliseconds, for reading from the current Marathon leader.",
+          "type" : "integer",
+          "default" : 10000
+        },
+        "local-port-max" : {
+          "description" : "Max port number to use when assigning globally unique service ports to apps.",
+          "type" : "integer",
+          "default" : 20000
+        },
+        "local-port-min" : {
+          "description" : "Min port number to use when assigning globally unique service ports to apps.",
+          "type" : "integer",
+          "default" : 10000
+        },
+        "logging-level" : {
+          "description" : "Set logging level to one of: off, error, warn, info, debug, trace, all",
+          "type" : "string"
+        },
+        "marathon-store-timeout" : {
+          "description" : "(deprecated) Maximum time, in milliseconds, to wait for persistent storage operations to complete. This option is no longer used and will be removed in a later release.",
+          "type" : "integer"
+        },
+        "master" : {
+          "description" : "The URL of the Mesos master",
+          "type" : "string",
+          "default" : "zk://master.mesos:2181/mesos"
+        },
+        "max-apps" : {
+          "description" : "The maximum number of applications that may be created.",
+          "type" : "integer"
+        },
+        "max-tasks-per-offer" : {
+          "description" : "Maximum tasks per offer. Do not start more than this number of tasks on a single offer.",
+          "type" : "integer",
+          "default" : 1
+        },
+        "mesos-authentication-principal" : {
```

Comment:
I can not set a default value here.
Or do you think this should be part of the description?